src/content/docs/en/guides/cms/payload.mdx

```diff
@@ -6,7 +6,222 @@ stub: true
+
+```astro title="src/collections"
+import { CollectionConfig } from "payload/types";
+
+const Posts: CollectionConfig = {
+  slug: "posts",
+  admin: {
+    useAsTitle: "title",
+  },
+  access: {
+    read: () => true,
+  },
+
+  fields: [
+    {
+      name: "title",
+      type: "text",
+      required: true,
+    },
+    {
+      name: "content",
+      type: "text",
+      required: true,
+    },
+    {
+      name: "slug",
+      type: "text",
+      required: true,
+    },
+  ],
+};
+
+export default Posts;
+```
+
+This creates a "Posts" collection which requires the following : `title`, `content`, `slug`. You will use this collection to manage your blog posts. The colllection is publicly available, so you shouldn't put any sensitive info inside.
+
+You then need to import and add it to the available collections in the `payload.config.ts` file.
+
+```astro title="src/payload.config.ts"
+import { buildConfig } from "payload/config";
+import path from "path";
+
+import Users from "./collections/Users";
+import Posts from "./collections/Posts";
+
+export default buildConfig({
+  serverURL: "http://localhost:3000",
+  admin: {
+    user: Users.slug,
+  },
+  collections: [Users, Posts],
+  typescript: {
+    outputFile: path.resolve(__dirname, "payload-types.ts"),
+  },
+  graphQL: {
+    schemaOutputFile: path.resolve(__dirname, "generated-schema.graphql"),
+  },
+});
+```
+
+If you then go into the PayloadCMS Dashboard, you should see a new collection called "Posts" appear next to the "Users" collection. Enter the "Posts" collection and create a new post. After saving it, you will notice the API URL appear in the bottom right corner.
+
+If not already running, start the server using `npm run dev` again.
+
+Open `http://localhost:3000/api/posts` in your browser. You should see a JSON file containing the post you have created as an object.
+
+:::tip
+By default, both Astro and PayloadCMS will use the port 3000. You might want to change the PayloadCMS port in the `src/server.ts` file. Don't forget to update the `serverURL` in `src/payload.config.ts` as well.
+:::
+
+```astro title="http://localhost:3000/api/posts"
+{
+   "docs":[
+      {
+         "id":"64098b16483b0f06a7e20ed4",
+         "title":"Astro & PayloadCMS Title 🚀",
+         "content":"Astro & PayloadCMS Content",
+         "slug":"astro-payloadcms-slug",
+         "createdAt":"2023-03-09T07:30:30.837Z",
+         "updatedAt":"2023-03-09T07:30:30.837Z"
+      }
+   ],
+   "totalDocs":1,
+   "limit":10,
+   "totalPages":1,
+   "page":1,
+   "pagingCounter":1,
+   "hasPrevPage":false,
+   "hasNextPage":false,
+   "prevPage":null,
+   "nextPage":null
+}
+```
+
+### Fetching Data
+
+Fetch your PayloadCMS data through your site's unique REST API URL and the route for your content. (By default, PayloadCMS will mount all routes through `/api`.) Then, you can render your data properties using Astro's `set:html=""` directive.
```

Comment:
Have we mentioned what the site's unique REST API URL is, or where to find it? Is this just depending on the port used? `localhost:portnumber/api/posts`?